### PR TITLE
assigment2 attempt 1

### PR DESCRIPTION
**Pull request by sfabriece**

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -2,9 +2,9 @@
 
 var Hapi = require('hapi');
 var Hoek = require('hoek');
+var Version = require('./version');
 var Package = require('../package.json');
```
Comment (chyld) [inline lib/index.js]: you're not using the Package module in this file

File: lib/index.js [hunk 2]
```diff
@@ -15,16 +15,11 @@ internals.init = function () {
...
-        path: '/version',
-        config: {
-            description: 'Returns the version of the server',
-            handler: function (request, reply) {
-
-                return reply({ version: Package.version });
-            }
-        }
+    server.register(Version, function(err){
+       if(err){
```
Comment (chyld) [inline lib/index.js]: use Hoek for error checking

File: lib/version.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+//Load modules
+var Package = require('../package.json');
+
+//Declare internals
+var internals = {};
```
Comment (chyld) [inline lib/version.js]: you've declared internals, but are not using it

Comment (hueniverse) [inline lib/version.js]: That's the correct style. Always declare `internals`.

Comment (AdriVanHoudt) [inline lib/version.js]: Even when not used? Seems redundant. Also it will throw a linting error for unused vars...

File: lib/index.js (hunk 1 above)
Comment (sfabriece) [inline lib/index.js]: The reply is getting the Version number from Package.

**Pull request closed without merging**